packages/cli/src/commands/build-docs/utils.ts

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { createStore, Redoc } from 'redoc';
+import { parseYaml, findConfig, Config } from '@redocly/openapi-core';
+import { renderToString } from 'react-dom/server';
+import { ServerStyleSheet } from 'styled-components';
+import { compile } from 'handlebars';
+import { join } from 'path';
+import { existsSync, lstatSync, readFileSync } from 'fs';
+
+import type { BuildDocsOptions } from './types';
+
+export function getObjectOrJSON(
+  options: string | Record<string, unknown>
+): JSON | Record<string, unknown> | Config {
+  const configFile = findConfig();
```

Comment:
I would call find `findConfig()` only in `default` case because this is only case where we use it

Reply:
I prefer it too. But eslint config has the rule `no-case-declarations`. I disabled it for this case.
But I see in the code we have other cases with ignoring this rule.
What do you think, should we disable this rule in the future? Does that make sense?

Reply:
understand it